### DISCUSSION TITLE
Does this have support for Woocommerce or would we need to add this functionality ourselves?

### DISCUSSION THREAD

**Discussion by JordanBettridge** [colbyfayock/next-wordpress-starter#387] [Q&A]
Does this have support for Woocommerce or would we need to add this functionality ourselves?
I am looking for a starter that has Woocommerce enabled by Default so I can make a NextJS website with woocommerce

**Comment by colbyfayock** [comment]
hey, unfortunately it doesn't have out of the box woocommerce support
